### PR TITLE
Janimommis

### PR DESCRIPTION
**Pull request by F-OS**
Gives MOMMIs soap and light replacers so they can clean stuff up, they can still pick up soap and light replacers, this just makes it easier. 
Made 100% with the web editor so I take full responsibility if this crashes and burns, shouldn't though.
[tweak]
:cl:
 * tweak: MOMMIs now have light replacers and soap

### CODE REVIEW

File: code/modules/mob/living/silicon/mommi/mommi_modules.dm [hunk 1]
```diff
@@ -25,9 +25,11 @@
 	modules += new /obj/item/weapon/extinguisher/foam(src)
 	modules += new /obj/item/device/rcd/rpd(src)
 	modules += new /obj/item/device/rcd/tile_painter(src)
+	modules += new /obj/item/device/lightreplacer/borg(src)
 	modules += new /obj/item/blueprints/mommiprints(src)
 	modules += new /obj/item/device/material_synth/robot/mommi(src)
 	modules += new /obj/item/device/holomap(src)
+	modules += new /obj/item/weapon/soap/syndie(src)
```
Comment (jknpj) [inline code/modules/mob/living/silicon/mommi/mommi_modules.dm]: >non-syndie mommies using syndie soap

:thinking:

Comment (F-OS) [reply code/modules/mob/living/silicon/mommi/mommi_modules.dm]: really makes you think

**Comment by iFlashYou** [comment]
cult nerf

**Comment by F-OS** [comment]
@iFlashYou I think most mommis would have the good judgement not to clean during cult rounds if it's obvious, they can still clean right now by just grabbing soap from maint/perma.

**Comment by SonixApache** [comment]
even more powercreep for crabs
even less reason to play janitor

File: code/modules/mob/living/silicon/mommi/mommi_modules.dm [hunk 2]
```diff
@@ -76,7 +78,6 @@
 	modules += new /obj/item/device/rcd/borg/engineering(src)
 	modules += new /obj/item/device/instrument/instrument_synth(src)
 	modules += new /obj/item/device/rcd/borg/rsf/soviet(src)
-	modules += new /obj/item/weapon/soap/syndie(src)
```
Comment (jknpj) [inline code/modules/mob/living/silicon/mommi/mommi_modules.dm]: >syndie mommis without syndie soap

:thinking:

**Comment by Lunstra** [comment]
As a leading member of the MoMMI metaclub I came here to say "stop buffing crabs holy shit they're already the single strongest entity on the entire station what the FUCK is wrong with you people?" Seriously, there's a reason we've hamstrung them with an unchangable lawset that forbids them from involving themselves with the crew.

**Comment by Caboozles** [comment]
MoMMis equipped with a street sweeper to prevent station damage when?

**Comment by FudgePucker** [comment]
By the time you get the engine set up, miners should be bringing back materials. Just print out a chem sprayer and fill it with space cleaner

**Comment by FudgePucker** [comment]
Maybe an upgrade, so that way you dont immediately get it (require uranium or diamonds to prevent "LMAO vent crawl to vault" from happening.)

File: code/modules/mob/living/silicon/mommi/mommi_modules.dm (hunk 2 above)
Comment (F-OS) [reply code/modules/mob/living/silicon/mommi/mommi_modules.dm]: Really does make you think a lot.

**Comment by F-OS** [comment]
I understand the concerns behind this nerfing cult, however I believe this is a administrative issue rather then a code one, allowing mommis to have soap and light replacers will allow them to clean fire-damaged rooms, handle that power surge event that knocks the lights out, and just generally help out. 

Considering most of these supplies are obtainable within 2 minutes for a skilled mommi, worrying about this being a possible balance issue doesn't seem smart.

**Comment by Allspice** [comment]
1. "a skilled mommi" can do it, but I suspect most do not, to say nothing of the entire mome population
2. the soap and replacer would be in your internal modules instead of your claw/toolbox/whatever

**Comment by jknpj** [comment]
you forgot to re-add the syndie soap to soviet mommis

**Comment by amikawaii** [comment]
Crabs dont need more powah creep. Janiborgs and janitor can hold this niche with no problem. Additionally all these items are just a crate away. Keep mommis engineering only, not custodial.

**Comment by bathosbathtime** [comment]
Make them find or make their own soap.  Light replacer makes sense for a maintenance drone thing, I guess.

**Comment by yclatious** [comment]
What the fuck is this. What in the GODDAMN is this level of beyond baloney bullshit? Where in the perdiferous, perilous pervesity of your putrid pile of paste you call a brain is the bundle of frazzled wires you call neurons that gave you this shit idea?

Because you need those destroyed, demolished, detained and deleted. You already have soap in your den, and you have soap in the mechanics lockers, which they often dont pick up. You also have cleaning supplies around and you can make ammonia water with plenty of ease and get a sprayer too. Even a chem sprayer with your tech ability, able to clean entire fucking hallways.

No. Any colab I see merging this has the big fucking gay.

**Comment by Intigracy** [comment]
There should be things that are restricted to mommis that actually know what they're doing

Soap is honestly one of them

**Comment by Shadowmech88** [comment]
Should MoMMIs just be made capable of literally anything? After all, the *ability* to do any job on the station doesn't *necessarily* lead to involvement and breaking of laws.
But if not, if there is a line on what MoMMIs should be capable of both flavor and mechanics-wise, then I'm pretty sure this crosses it. I think even the microwave and bartending machines PR crossed it.
Because if those don't, then what would?

**Comment by FudgePucker** [comment]
This PR is a bit much I agree.
I don't think the microwave one did, just because they had zero ability to do those things for no reason.
This PR on the other hand gives them objects to make their lives easier, even though they LITTERALLY CAN JUST PICK UP SOAP AND USE IT ALREADY.
Stick a light replacer and some soap in a toolbox, bring the toolbox with you. If you wanted easy access to soap in your inventory you should play janitor or Janiborg.

**Comment by Fruktlimpa** [comment]
The microwave PR was the start of a slip slide down the slippery slope and this PR is an example of that slope.
MoMMI's are already better than Engineers in pretty much every way, they don't need to also be better than Janitors, especially considering you can just pick up soap/make a chem sprayer/steal a spray from medbay.
Soap is bad enough but the biggest fuck you to Janitors would be giving MoMMI's a light replacer, fucking hell.

There is no reason to make MoMMI's even better than they already are and they are meant to be Engineering robots not Do-everything-robots, even if they can already do most of everything. It doesn't have to be even easier.

**Comment by bathosbathtime** [comment]
They start with soap in their den and having only one hand with limited storage for non-modules should be something they have to work around if they want to clean/do anything really

**Comment by amikawaii** [comment]
And thats a good thing

**Comment by F-OS** [comment]
Not enough popular support.

**Pull request closed without merging**